Assets/MRTK/Providers/LeapMotion/Editor/ConfigurationChecker/LeapMotionConfigurationChecker.cs

```diff
@@ -23,14 +23,18 @@ static class LeapMotionConfigurationChecker
         // True if the Leap Motion Core Assets are in the project.
         private static bool isLeapInProject = false;
 
-        // The currently supported Leap Core Assets version number.
-        private static string leapCoreAssetsVersionNumber = "4.4.0";
+        // The current supported Leap Core Assets version numbers.
+        private static string[] leapCoreAssetsVersionsSupported = new string[] { "4.4.0", "4.5.0"};
+
+        // The current Leap Core Assets version in this project
+        private static string currentLeapCoreAssetsVersion = "";
 
         // The path difference between the root of assets and the root of the Leap Motion Core Assets.
         private static string pathDifference = "";
 
         // Array of paths to Leap Motion testing directories that will be removed from the project.
         // Make sure each test directory ends with '/'
+        // These paths only need to be deleted if the Leap Core Assets version is 4.4.0
```

Comment:
It looks like we still try to delete these (at least, I can't see the path where the `Delete` method is excluded). Will it just fail because none of the paths exist?

Reply:
Yes, it will fail because the paths do not exist